src/Plugin.php

```diff
@@ -88,9 +88,15 @@ public static function init() {
     add_action('woocommerce_process_product_meta', __NAMESPACE__ . '\WooCommerce::woocommerce_process_product_meta');
     add_action('woocommerce_save_product_variation', __NAMESPACE__ . '\WooCommerce::woocommerce_save_product_variation', 10, 2);
 
-// Adds woocommerce specific settings.
+    // Adds woocommerce specific settings.
     add_filter('woocommerce_get_settings_shop_standards', __NAMESPACE__ . '\WooCommerce::woocommerce_get_settings_shop_standards');
 
+    // Allow ajax requests to specified functions.
+    add_action('wp_ajax_is_existing_gtin', __NAMESPACE__ . '\WooCommerce::ajax_is_existing_gtin');
+
+    // Enqueues admin plugin scripts.
+    add_action('admin_enqueue_scripts', __CLASS__ . '::admin_enqueue_scripts');
+
     Seo::init();
     WooCommerceSalutation::init();
 
```

Comment:
Just to understand: what is this for (in comparison to the distinction between `Plugin.php` and   `Admin.php`)?

Reply:
`Plugin::init()` is invoked at all times. Everything below the `return is_admin()` in there is only executed in the front-end, not in the administration interface.

`Admin::init()` is only invoked in the administration interface.

Reply:
https://github.com/netzstrategen/wordpress-shop-standards/blob/299f984cc92c6920fc41e2724356ee27ef41642a/src/Plugin.php#L81-L89	

I don't want to open up a huge discussion here, but wouldn't lines like these need to be placed in `Admin.php` as well? They are never needed in the front-end imho.